homeassistant/components/icloud/const.py

```diff
@@ -1,7 +1,7 @@
 """iCloud component constants."""
 
 DOMAIN = "icloud"
-TRACKER_UPDATE = f"{DOMAIN}_tracker_update"
+SERVICE_UPDATE = f"{DOMAIN}_update"
```

Comment:
I'd call it `DEVICE_UPDATE`.

Reply:
Indeed, I will add more sensors in the future, which does not refer to a device, something like remaining iCloud Drive space, calendar ...

All of them will use this same signal, that's why I called it like this.

Also I saw some `SERVICE_UPDATE` in the code.

Reply:
Ok.

Reply:
The signal refers to the end of the service queries and the end of the `IcloudAccount` update.